From 1d8f4bb2528c1b2db6e93168077c414963d1b3be Mon Sep 17 00:00:00 2001
From: Otto Liljalaakso <otto.liljalaakso@iki.fi>
Date: Mon, 17 Mar 2025 22:36:58 +0200
Subject: [PATCH] Revert "New style license metadata"

This reverts commit 7c53d148de1e5722271c7d9c3249746a25b29bf5 in order
retain compatibility with Flit versions older than 3.11 as present in
Fedora < 43.

diff --git a/pyproject.toml b/pyproject.toml
index 367998b..74a8800 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,8 +5,7 @@ readme = "README.md"
 authors = [
     {name = "Antti Ajanki", email = "antti.ajanki@iki.fi"}
 ]
-license = "GPL-3.0-or-later"
-license-files = ["COPYING"]
+license = {text = "GPLv3+"}
 dependencies = [
     "requests",
     "lxml",
@@ -17,6 +16,7 @@ classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Environment :: Console",
     "Intended Audience :: End Users/Desktop",
+    "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
     "Operating System :: OS Independent",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
@@ -46,7 +46,7 @@ yle-dl = "yledl.yledl:main"
 [project.urls]
 Home = "https://aajanki.github.io/yle-dl/"
 Source = "https://github.com/aajanki/yle-dl/"
-changelog = "https://github.com/aajanki/yle-dl/blob/master/ChangeLog"
+changelog = "https://raw.githubusercontent.com/aajanki/yle-dl/master/ChangeLog"
 issues = "https://github.com/aajanki/yle-dl/issues"
 
 [tool.flit.module]
@@ -73,5 +73,5 @@ quote-style = "single"
 disable_error_code = "import-untyped"
 
 [build-system]
-requires = ["flit_core >=3.11,<4"]
+requires = ["flit_core >=3.9,<4"]
 build-backend = "flit_core.buildapi"
-- 
2.48.1

